test/sql/order_by_queries_sql_test.cpp

```diff
@@ -166,5 +163,74 @@ TEST_F(OrderByQueriesSQLTests, OrderByWithoutColumnsAndLimitTest) {
   auto txn = txn_manager.BeginTransaction();
   catalog::Catalog::GetInstance()->DropDatabaseWithName(DEFAULT_DB_NAME, txn);
   txn_manager.CommitTransaction(txn);
-}}  // namespace test
+}
+
+TEST_F(OrderByQueriesSQLTests, OrderByStar) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  CreateAndLoadTable();
+
+  std::vector<ResultType> result;
+  std::vector<FieldInfoType> tuple_descriptor;
+  std::string error_message;
+  int rows_changed;
+  std::unique_ptr<optimizer::AbstractOptimizer> optimizer(
```

Comment:
Sure, however, the code changes are applied in the `SimpleOptimizer`. How would the test code execute this code?